language/de.json

```diff
@@ -1,292 +1,321 @@
 {
-  "semantics": [
+  "semantics":[
     {
-      "fields": [
+      "fields":[
         {
-          "field": {
-            "fields": [
+          "field":{
+            "fields":[
               {
-                "field": {
-                  "fields": [
-                    {},
-                    {},
-                    {},
-                    {},
-                    {},
+                "field":{
+                  "fields":[
                     {
-                      "label": "Kommentare",
-                      "description": "Die Kommentare werden angezeigt, wenn der Nutzer die vorgeschlagenen Antworten für alle Schaubilder anzeigt."
+                      
                     },
                     {
-                      "label": "Kommentare immer anzeigen"
+                      
                     },
                     {
-                      "label": "Undurchlässigkeit des Hintergrunds"
+                      
                     },
                     {
-                      "label": "Als Button anzeigen"
+                      
                     },
                     {
-                      "label": "Titel"
+                      
                     },
                     {
-                      "label": "Gehe zu Schaubild"
+                      "label":"Kommentare",
```

Comment:
It seems that in the editor, not everything is translated, e. g. most hover texts for the upper icon bar or the phrases connected to the keyword list ("Schlagwortliste" in German) although they seem to be contained in the language file. I have not had a look at the source code yet to figure out what might be the problem.